modules/kuberay-cluster/main.tf

```diff
@@ -167,3 +168,15 @@ resource "kubernetes_network_policy" "kuberay-cluster-allow-network-policy" {
     policy_types = ["Ingress"]
   }
 }
+
+# Assign resource quotas to Ray namespace to ensure that they don't overutilize resources
+resource "kubernetes_resource_quota" "ray_namespace_resource_quota" {
```

Comment:
this resource block create quota on overall namespace.. and it will impact other dependent deployment and services.

Either add the match expression so this applies to only selected resources..

or take this out of this module..and add at namespace level.

Reply:
There is no match expression for resource quotas. They always apply at the namespace level though you can scope them to priority classes (which we haven't defined). These defaults are left intentionally high so that they won't interfere with anything in our demo. If you're concerned, the right thing to do is to split Ray out into a separate namespace.

Reply:
okay agree.. lets move this out of kube-ray-cluster module.. and place it with rag/main.tf so it is configurable at that level.. rather then configuring it while calling kuberay-cluster module.

Reply:
This isn't for RAG. This is for a blog post about Ray hardening that I'm writing. I'm testing against RAG because RAG is using Ray.